code/modules/antagonists/changeling/powers/mutations.dm

```diff
@@ -291,47 +277,76 @@
 				var/mob/living/carbon/C = L
 				switch(firer.a_intent)
 					if(INTENT_HELP)
-						C.visible_message("<span class='danger'>[L] is pulled by [H]'s tentacle!</span>","<span class='userdanger'>A tentacle grabs you and pulls you towards [H]!</span>")
+						C.visible_message("<span class='danger'>[L] is pulled to their feet towards [H]!</span>","<span class='userdanger'>A tentacle grabs you and pulls you up towards [H]!</span>")
 						add_attack_logs(H, L, "[H] pulled [L] towards them with a tentacle")
 						C.throw_at(get_step_towards(H,C), 8, 2)
-						return 1
+						C.AdjustParalysis(-2 SECONDS)
+						C.AdjustStunned(-4 SECONDS)
+						C.AdjustWeakened(-4 SECONDS)
+						C.AdjustKnockDown(-4 SECONDS)
+						C.adjustStaminaLoss(-25)
+						return TRUE
 
 					if(INTENT_DISARM)
 						var/obj/item/I = C.get_active_hand()
 						if(I)
 							if(C.drop_item())
 								C.visible_message("<span class='danger'>[I] is yanked out of [C]'s hand by [src]!</span>","<span class='userdanger'>A tentacle pulls [I] away from you!</span>")
-								add_attack_logs(src, C, "[src] has grabbed [I] out of [C]'s hand with a tentacle")
+								add_attack_logs(H, C, "[src] has grabbed [I] out of [C]'s hand with a tentacle")
 								on_hit(I) //grab the item as if you had hit it directly with the tentacle
-								return 1
-							else
-								to_chat(firer, "<span class='danger'>You can't seem to pry [I] out of [C]'s hands!</span>")
-								add_attack_logs(src, C, "[src] tried to grab [I] out of [C]'s hand with a tentacle, but failed")
-								return 0
-						else
-							to_chat(firer, "<span class='danger'>[C] has nothing in hand to disarm!</span>")
-							return 0
+								return TRUE
+							to_chat(firer, "<span class='danger'>You can't seem to pry [I] out of [C]'s hands!</span>")
+							add_attack_logs(H, C, "[src] tried to grab [I] out of [C]'s hand with a tentacle, but failed")
+						C.visible_message("<span class='danger'>[C] is knocked over by [src]!</span>","<span class='userdanger'>A tentacle hits you in the chest and knocks you over!</span>")
+						add_attack_logs(H, C, "[src] knocked over [C] with a tentacle")
+						C.KnockDown(2 SECONDS) //Not useless with antidrop.
+						return TRUE
 
 					if(INTENT_GRAB)
-						C.visible_message("<span class='danger'>[L] is grabbed by [H]'s tentacle!</span>","<span class='userdanger'>A tentacle grabs you and pulls you towards [H]!</span>")
-						add_attack_logs(H, C, "[H] grabbed [C] with a changeling tentacle")
-						C.throw_at(get_step_towards(H,C), 8, 2, callback=CALLBACK(H, TYPE_PROC_REF(/mob, tentacle_grab), C))
-						return 1
+						C.visible_message("<span class='danger'>[L] is entangled by [H]'s tentacle!</span>","<span class='userdanger'>A tentacle grabs you and wraps around your legs!</span>")
+						add_attack_logs(H, C, "[H] imobilised [C] with a changeling tentacle")
```

Comment:
```suggestion
						add_attack_logs(H, C, "imobilised with a changeling tentacle")
```
The names already exist in the log. This makes it easier to search for as well since it is always the same message